src/lib/platform/OSXScreen.mm

```diff
@@ -811,6 +811,8 @@
 			IORegistryEntrySetCFProperty(entry, CFSTR("IORequestIdle"), kCFBooleanFalse);
 			IOObjectRelease(entry);
 		}
+		IOPMAssertionID assertionID; 
+		IOPMAssertionDeclareUserActivity(CFSTR(""), kIOPMUserActiveLocal, &assertionID);
```

Comment:
According to the documentation we should fill in the AssertionName field (first parameter).

Reply:
What name would you suggest? Or should we just put something like 'INPUT-LEAP_WAKE'?